### ISSUE TITLE
The specified module could not be found

### ISSUE THREAD

**Issue by jbren77** [gtkashi/nameless#1]
First off can I just say THANK YOU to you and the contributors for making this!

Sorry if this is on my end, I'm kind of a noob at loading custom addons to windower.
I have the entire nameless folder from here put into "nameless" in my addons folder.

When I run //lua l nameless I get this error in-game:

`nameless: Lua runtime error: nameless/nameless.lua:16: The specified module could not be found`

It looks like it's pointing to something with the EntityFlagChanger. I've tried putting the dll in a few different locations but it hasn't fixed it. I have a pretty basic windower installation, just InfoBar and Config.

Also this is on Windows 11.

**Comment by gtkashi** [comment]
Curious. How's your folder structure setup? For me it's simply ".../Windower/addons/nameless/" and that's where nameless.lua and EntityFlagChanger.dll are stored.

**Comment by jbren77** [comment]
I think it's on my end to be honest. I was getting the error on my laptop, but I tried it on my desktop PC (win 11) and it works perfectly fine! Then I tried it again on my laptop and it started working lol! Maybe there was an issue with the download or I messed something else up. Thanks again for making this, I've always worried about burn in playing this game on my CRT, but not anymore!
![Screenshot 2022-12-21 132725](https://user-images.githubusercontent.com/112284185/209005664-c58ec6dd-0b58-4905-a5ba-da81372764fb.png)
